### DISCUSSION TITLE
[v3] How can I hide the page in the link when I use WithPagination?

### DISCUSSION THREAD

**Discussion by devhau** [livewire/livewire#6681] [1. Help]
How can I hide the page in the link when I use WithPagination?
![image](https://github.com/livewire/livewire/assets/50167033/67757fd4-f15b-4c37-9795-ce203fbdb413)
Thanks

**Comment by N1ebieski** [comment]
There is no such option at the moment, I think.

Temporarily, I did a trick to terminate the method [SupportPagination::ensurePaginatorIsInitialized](https://github.com/livewire/livewire/blob/main/src/Features/SupportPagination/SupportPagination.php#L73):

```php
public function mount(): void
{
   $this->paginators['page'] = 1;
}
```

**Reply by devhau** [reply]
Yes, Thanks!
I try to find another solution.
